styles/style.css

```diff
@@ -267,3 +272,111 @@ footer p a {
         text-align: center;
     }
 }
+
+table {
+    width: 100%;
+    border-spacing: 0;
+    border: 1px solid #D9D9E8;
+    border-radius: 4px 4px 0px 0px;
+}
+
+tr th,
+tr td{
+    border-bottom: 1px solid #D9D9E8;
+    box-sizing: border-box;
+}
+
+thead tr,
+tfoot tr{
+    background-color: #EEF9FF;
+}
+
+thead th {
+    text-align: start;
+}
+
+th, td {
+    padding: 15px;
+}
+
+tbody th:nth-child(1) {
+    text-align: start;
+}
+
+tbody td {
+    color: black;
+}
+
+tfoot th {
+    border: none;
+    text-align: start;
+}
+
+tfoot td {
+    border: none;
+}
+
+@media
+only screen and (max-width: 760px),
+(min-device-width: 768px) and (max-device-width: 1024px)  {
+    table, thead, tbody, th, td, tr, tfoot {
+        display: block;
+        border: none;
+    }
+
+    tbody tr:not(:last-child) {
+        margin-bottom: 10px;
+    }
+
+    thead tr {
+        position: absolute;
+        top: -9999px;
+        left: -9999px;
+    }
+
+    tr { border: 1px solid #D9D9E8; }
+
+    td {
+        border: none;
+        border-bottom: 1px solid #D9D9E8;
+        position: relative;
+        padding-left: 40%;
+    }
+
+    td:last-child {
+        border: none;
+    }
+
+    td:before {
+        position: absolute;
```

Comment:
Using {white-space: nowrap} with a fixed width could end with text overlapping boundaries. Try adding more text to cells and see on mobile. Adding {overflow: hidden; text-overflow: ellipsis} should help.